### DISCUSSION TITLE
about multiple store

### DISCUSSION THREAD

**Discussion by 0xMALVEE** [pmndrs/zustand#2325] [Q&A] [answered]
So i have two stores in 2 files using create store. but in doc I see there is some slice pattern why we even need that slice pattern for me this just works fine can I not use this in production will there be any problems if I use create() multiiple times many storesl ? 
The one like multiple store
![image](https://github.com/pmndrs/zustand/assets/35369563/cc3e9366-02fd-4901-af71-1232594f895f)
![image](https://github.com/pmndrs/zustand/assets/35369563/73047581-92da-494b-aa38-12d8dde06540)
in docs
![image](https://github.com/pmndrs/zustand/assets/35369563/63036eac-74a7-457e-84ae-f64796a7ff14)

**Comment by dbritto-dev** [comment, marked as answer]
@0xMALVEE it depends on your needs, if there's no connection between the two stores is OK to have multiple stores but if there's a connection between them the slice pattern is the right way to implement them
